packages/chart/src/components/BarChart/helpers/index.ts

```diff
@@ -76,8 +83,17 @@ export const getBarConfig = ({ bar, defaultBarHeight, defaultBarWidth, config, i
     // determine label width in pixels & check if it fits to the bar width
     const labelWidth = getTextWidth(barLabel, `normal ${barWidth / 2}px sans-serif`)
     const labelFits = labelWidth < barWidth && barWidth > 10
-    return labelFits && isVertical ? label : !isVertical ? label : ''
+    if (config.isLollipopChart) {
+      return label
+    } else {
+      return labelFits && isVertical ? label : !isVertical ? label : ''
+    }
   }
 
   return { barWidthHorizontal, barHeight, isSuppressed, showMissingDataLabel, showZeroValueDataLabel, getBarY, getAbsentDataLabel }
 }
+
+export const testZeroValue = value => {
```

Comment:
Remember to write tests for helper functions. This could also be beneficial for other packages and could potentially live in core.